core/__tests__/modules/configWriter.ts

```diff
@@ -0,0 +1,354 @@
+import fs from "fs";
```

Comment:
I chose to put everything related to writing files here. Most of this should probably be in the model specs, but it felt more efficient to cover it while building out a series of objects all related to the same tasks. I chose efficiency of the test suite over spreading these things out.